projects/vdk-core/src/vdk/internal/builtin_plugins/termination_message/file_util.py

```diff
@@ -23,7 +23,7 @@ def append_to_file(self, message):
             return
 
         try:
-            with open(self.filename, "a") as file:
+            with open(self.filename, "w") as file:
                 file.write(message)
```

Comment:
Once per data job run. 
But If you use StandaloneDataJob() inside a normal data job. Then It can be called more than once.

Which is exactly what is happening.

Reply:
It's seems ambiguously/not documented; 
In that train of thought, is there any core hook that will be evaluated once at the very end of the job..?

The use-cases I am asking about, are for example finalizing Telemetry client, or overwriting the exit status in case of User error.